src/api/chat-post-with-weather-info.js

```diff
@@ -0,0 +1,84 @@
+const https = require("https");
+const TOKEN = process.env.LINE_ACCESS_TOKEN;
+
+const chatPostWithWeatherInfo = function(event) {
+     // Text and type information
+    const replyToken = event.replyToken;
+    const messageType = event.message.type;
+    const messageText = event.message.text;
+
+    if (messageType !== "text") {
+        return;
+    }
+    
+    // URL is sent to Open weather API
+    const APIKEY = process.env.OPEN_WEATHER_API_KEY;
+    const unit = "metric";
+    const url = "https://api.openweathermap.org/data/2.5/forecast?q=" + messageText + "&appid=" + APIKEY + "&units=" + unit;
+
+    // Set information
+    https.get(url, function(response) {
+        response.on("data", function(data) {
+            const weatherData = JSON.parse(data);
+            console.log(weatherData);
+
+            // Set four-day weather forecast information to message array
+            const message = [];
+            for (let i = 0; i < 32; i += 8) {
```

Comment:
where 32 and 8 come from?

Reply:
Eight weather data is sent per day from weatherAPI and this bot uses 4 days of data. So that's why there is those number.